platform/migration/platform-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+    xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <bean id="ddfHomeFile" class="java.io.File">
+        <argument value="${ddf.home}" />
+    </bean>
+
+    <bean id="ddfHomeUri" class="java.net.URI" factory-ref="ddfHomeFile"
+        factory-method="toURI" />
+
+    <bean id="ddfHome" class="java.nio.file.Paths" factory-method="get">
+        <argument ref="ddfHomeUri" />
+    </bean>
+
+    <bean id="platformMigratable" class="org.codice.ddf.migration.impl.PlatformMigratable">
+        <argument ref="ddfHome" />
```

Comment:
Do you think we should pass `ddfHome` in the `export()` method instead of having implementers inject it?